for reviewers

SenaitEkubai opened the pull request.

SenaitEkubai committed:
added my name and username

SenaitEkubai committed:
added a link to tvmaze

SenaitEkubai committed:
added style to the page

SenaitEkubai committed:
done level 100

for each episode, i have added a name, season, season number and season image

SenaitEkubai committed:
Done level 200

SenaitEkubai committed:
Done level 300

SenaitEkubai committed:
Done level 300 level

SenaitEkubai committed:
added some styling

SenaitEkubai committed:
Done level 350

SenaitEkubai committed:
added inage for no image shows

SenaitEkubai committed:
Done level 400

SenaitEkubai committed:
Done half of level 500

SenaitEkubai committed:
filter not working

SenaitEkubai committed:
fixed filter

SenaitEkubai committed:
Merge pull request #1 from SenaitEkubai/Senait

merge the senait branch

SenaitEkubai committed:
still working on level 500

SenaitEkubai committed:
Merge pull request #2 from SenaitEkubai/Senait

done some changes please have a look

SenaitEkubai committed:
level 500 completed

SenaitEkubai committed:
Merge pull request #3 from SenaitEkubai/Senait

level 500 completed

SenaitEkubai committed:
added a catch

SenaitEkubai committed:
adjusted styles

SenaitEkubai committed:
adjusted html

SenaitEkubai committed:
Merge pull request #4 from SenaitEkubai/Senait

adjustments for both html and css

SenaitEkubai committed:
adjustments

Adepeju18 reviewed (commented):
Hi Senait, well done you have done an amazing job, the website looks OK and you have carried out all the instructions and you managed to finished all the levels and I love the way you put instructions on each function of your code and lastly I couldn't found any bug ,but I will like you to work more on your CSS. Well done

SenaitEkubai committed:
added some style

SenaitEkubai committed:
made title clickable

when user clicks show title it fetches all episodes
and when user clicks episode title it opens tv maze tab for the episode

SenaitEkubai committed:
Merge pull request #6 from SenaitEkubai/Senait

implemented feedback

SenaitEkubai committed:
renamed a variable

SenaitEkubai committed:
renamed varialbe

SenaitEkubai committed:
Merge branch 'master' of https://github.com/SenaitEkubai/tv-show-dom-project

SenaitEkubai committed:
resolve

SenaitEkubai committed:
resolved some conflicts

SenaitEkubai committed:
fetch casting function in progress

SenaitEkubai committed:
Started level 900

SenaitEkubai committed:
Merge branch 'master' into Senait

SenaitEkubai committed:
Merge pull request #7 from SenaitEkubai/Senait

Senait

mikinet commented on style.css:
```diff
@@ -44,142 +20,192 @@ h1 {
+  }
 }
+/* media query for over 400px */
 @media only screen and (min-width: 400px) {
```
I like that you added media queries. It improves the aesthetic qualities of your site.
